File: 0263-ugly-number/0263-ugly-number.java
class Solution {
    public boolean isUgly(int n) {
          if(n<=0){
            return false;
        }
        int div[]= {2,3,5};
        for(int divisor:div){
            while(n%divisor==0){
                n/=divisor;
            }
        }        
        return n==1;
    }
}

File: 0263-ugly-number/0263-ugly-number.py
class Solution(object):
    def isUgly(self, n):
        if n <= 0:
            return False
        for divisor in [2, 3, 5]:
            while n % divisor == 0:
                n //= divisor
        return n == 1
        """
        :type n: int
        :rtype: bool
        """
        